python/foglamp/plugins/north/http_north/http_north.py

```diff
@@ -41,6 +41,12 @@
         'default': 'http://localhost:6683/sensor-reading',
         'order': '1'
     },
+    "source": {
```

Comment:
what code reads stats here?

Reply:
It is managed in the sending process. Configuration is here to allow the GUI to pick it up.